src/esp/bindings_js/bindings_js.cpp

```diff
@@ -132,12 +132,25 @@ bool isBuildWithBulletPhysics() {
 #endif
 }
 
+int castRay(Simulator& sim,
+            const Magnum::Vector3& pos,
+            const Magnum::Vector3& dir,
+            float maxDistance) {
+  esp::geo::Ray ray;
```

Comment:
Nice, but wouldn't it be better to just add some quick bindings for the ray and ray hit result? Tat would give the user more control and make it more similar to the rest of our API.

Reply:
This has been done, thanks!